crates/starknet-devnet/src/cli.rs

```diff
@@ -237,19 +241,26 @@ impl Args {
                 url: self.fork_network.clone(),
                 block_number: self.fork_block,
             },
-            disable_account_impersonation: self.disable_account_impersonation,
         };
 
         let RequestResponseLogging { log_request, log_response } =
             RequestResponseLogging::from_rust_log_environment_variable();
 
+        let restricted_methods = self.restricted_methods.as_ref().map(|methods| {
+            if methods.is_empty() {
+                DEFAULT_RESTRICTED_JSON_RPC_METHODS.iter().map(|s| s.to_string()).collect()
+            } else {
+                methods.clone()
+            }
```

Comment:
So there is no method validation? I.e. if the strings specified are actual method names?

Reply:
So if a user specifies `--restrictive-mode mintt dump load`, minting will still be possible, without the user knowing they made a mistake?

Reply:
yes

Reply:
https://github.com/0xSpaceShard/starknet-devnet-rs/commit/bd705fa24b8e143696bcc3a47149f48f610956be..11e8f9c22b556b9f2e2829d2fc3841bef30d9990
https://github.com/0xSpaceShard/starknet-devnet-rs/commit/c857060ecc91bdeff62ef0b99b88ca4bed8147fc
https://github.com/0xSpaceShard/starknet-devnet-rs/commit/adcb70e1ed27c75899435af2d9588d44a40e19d2
https://github.com/0xSpaceShard/starknet-devnet-rs/commit/0dd445aa576f6a2cea3fca83d84e196f0950ed0b
https://github.com/0xSpaceShard/starknet-devnet-rs/commit/11e8f9c22b556b9f2e2829d2fc3841bef30d9990